nlp/notebooks/nl2query/NER_flair.py

```diff
@@ -11,8 +11,10 @@ class NER_flair(NL2QueryInterface):
     def __init__(self, config: str = None):
         super().__init__(config)
         # start my NL2query engine
+        # Passing a config containing the local path to the model, otherwise it will download the model
+        self.model_file = self.config.get('model_file','ner-large') if config is not None else 'ner-large'
```

Comment:
Some suggestions:
- Use a fallback in the ConfigParser
- Check against `self.config` rather than `config` in case that the base class implement a default config loading.
- `is not None` is not required in a if 
I would get something like that :
```
default = "ner-large"
self.model_file = self.config.get("model_file", "ner-large", fallback=default) if self.config else default
```